### DISCUSSION TITLE
Using artifact-hub with nexus

### DISCUSSION THREAD

**Discussion by zerr0s** [artifacthub/hub#3920] [Q&A] [answered]
Hello,

Please forgive me for my bad english.

I'm trying to deploy atifact-hub with a nexus proxy. For all image downloading, everything is OK. But I'd like to overwrite the trivy-db download and it looks like the artifact-hub chart does not allow to modify the dburl or the command.

How should I do to allow trivy to get its db update without error please ?

Hope that someone could help ...

**Comment by tegioz** [comment]
Hi @zerr0s 👋

Maybe we could expose the `db-repository` parameter of the Trivy CLI tool in the Artifact Hub chart to make this configurable. By default Trivy tries to pull it from `ghcr.io/aquasecurity/trivy-db:2`. Please note that Trivy will also need to be able to pull images referenced in packages (charts, etc) in order to scan them.

```
% trivy image --help | grep db-repository
      --db-repository string        OCI repository to retrieve trivy-db from (default "ghcr.io/aquasecurity/trivy-db:2")
```

**Comment by tegioz** [comment, marked as answer]
Please note that the scanner component allows you to specify the [url of the Trivy server](https://artifacthub.io/packages/helm/artifact-hub/artifact-hub?modal=values-schema&path=scanner.trivyURL) to use to scan images. This would allow you to deploy your Trivy server separately and point Artifact Hub to it.

**Comment by zerr0s** [comment]
hello. Sorry for the late response. I've just deployed a custom trivy component. thanks for the help
